src/commands/set.js

```diff
@@ -68,39 +75,47 @@ const attemptWriteToFile = (variable, value) => {
 	return result;
 };
 
-const setBoolean = (variable, path) => value => {
+const setBoolean = dotNotation => value => {
 	if (!checkBoolean(value)) {
 		throw new ValidationError('Value must be a boolean.');
 	}
-
+	const dotNotationArray = dotNotation.split('.');
 	const newValue = value === 'true';
-	path.reduce(setNestedConfigProperty(newValue), config);
+	dotNotationArray.reduce(setNestedConfigProperty(newValue), config);
 
-	if (variable === 'testnet') {
+	if (dotNotation === 'api.testnet') {
 		liskAPIInstance.setTestnet(newValue);
 	}
 
-	return attemptWriteToFile(variable, value);
+	if (dotNotation === 'api.ssl') {
+		liskAPIInstance.setSSL(newValue);
+	}
+
+	return attemptWriteToFile(value, dotNotationArray);
 };
 
-const setString = (variable, path) => value => {
-	path.reduce(setNestedConfigProperty(value), config);
-	return attemptWriteToFile(variable, value);
+const setString = dotNotation => value => {
```

Comment:
The same as above, what is the reason of `dotNotation => value`?
would it be more simple if you do like `(dotNotation, value) =>`?